build.sbt

```diff
@@ -106,7 +106,10 @@ testScalastyle := scalastyle.in(Test).toTask("").value
 
 def getVersion(version: String): String = {
     version.split("\\.").toList match {
-        case major :: minor :: rest => s"$major.$minor.0" 
+        case major :: minor :: rest =>
+          if (rest.contains("0-SNAPSHOT")) s"$major.${minor.toInt - 1}.0"
```

Comment:
Actually, make sure you strip the SNAPSHOT and then apply condition. Because here it wont work when the version is actually 0.8.0, which should compare against 0.7.0 (here it would compare with 0.7.0) which wont wont work until 0.7.0 is actually published.